### PR TITLE
fix fio blogpost link

### PR DESCRIPTION
**Pull request by samuelbartels20**
Please review 
@jmhbnz

### CODE REVIEW

File: content/en/docs/v3.4/op-guide/hardware.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ etcd has a relatively small memory footprint but its performance still depends o
 
 Fast disks are the most critical factor for etcd deployment performance and stability.
 
-A slow disk will increase etcd request latency and potentially hurt cluster stability. Since etcd’s consensus protocol depends on persistently storing metadata to a log, a majority of etcd cluster members must write every request down to disk. Additionally, etcd will also incrementally checkpoint its state to disk so it can truncate this log. If these writes take too long, heartbeats may time out and trigger an election, undermining the stability of the cluster. In general, to tell whether a disk is fast enough for etcd, a benchmarking tool such as [fio][fio] can be used. Read [here][fio-blog-post] for an example.
```
Comment (jmhbnz) [inline content/en/docs/v3.4/op-guide/hardware.md:24]: Hey @samuelbartels20 please refer to my previous feedback, the `[fio-blog-post]` entry at the bottom of each doc should be where the link is updated. You've updated it for some pages this way but not all of them. 

Additionally - I notice each time you need to adjust a pull request you close and open a new pull request. This is a noisy way of doing things, it's much better to rebase/modify an existing pull request than close and open a new one each time.

If you have any questions or need any help please feel free to ping me in Kubernetes slack.

**Review by jmhbnz — CHANGES_REQUESTED** [review]

File: content/en/docs/v3.4/op-guide/hardware.md (hunk 1 above)
Comment (samuelbartels20) [reply content/en/docs/v3.4/op-guide/hardware.md:24]: @jmhbnz I appreciate your guidance. I will improve on these processes. I'm fixing the issue ASAP

**Review by jmhbnz — APPROVED** [review]
LGTM - Thanks @samuelbartels20

**Comment by samuelbartels20** [comment]
@jmhbnz Please all changes have been fixed.

**Pull request merged**